cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,153 @@
+# Log Level Extension
+
+
+## Attributes
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
```

Comment:
At least one of these attributes needs to be REQUIRED. Any reason they're both not?

Reply:
Each value can be used by itself. 
Maybe I need to adda constrant that saids that you MUST choose at-least one of them

Reply:
Yup I think you should do that, otherwise a CE w/o any knowledge of this spec is compliant, which will feel kind of odd.